docs/build/Build-for-ARTIK053-TizenRT.md

```diff
@@ -36,42 +36,20 @@ iotjs-tizenrt
   |  + config
   |      + tizenrt
   |          + artik05x
-  + TizenRT
 ```
 
-#### 2. Add IoT.js as a builtin application for TizenRT
+#### 2. Configure ROMFS for TizenRT
 
 ```bash
-cp iotjs/config/tizenrt/artik05x/app/ TizenRT/apps/system/iotjs -r
-cp iotjs/config/tizenrt/artik05x/configs/ TizenRT/build/configs/artik053/iotjs -r
-cp iotjs/config/tizenrt/artik05x/romfs.patch TizenRT/
+pushd ${IOTJS_ROOT_DIR}/deps/tizenrt
```

Comment:
a) `/deps/tizenrt` doesn't exist.
b) `cd ${IOTJS_ROOT_DIR}/...` seems ok. `pushd` doesn't work as a pair with `popd`, so that it leaves an unnecessary trace on the directory stack.